src/components/DataAccessRequestHeader.js

```diff
@@ -0,0 +1,69 @@
+import {filter, flatMap, flow, identity, join, map} from 'lodash/fp';
+import {div, span} from 'react-hyperscript-helpers';
+import {DataSet} from '../libs/ajax';
+import {useEffect, useState} from 'react';
+
+const style = {
+  'fontSize': 19,
+  'fontStyle': 'normal',
+  'margin': '10px 0 30px 56px',
+  'color': '#333333',
+  'fontWeight': 'normal',
+  'lineHeight': '27px',
+};
+
+const pipe = ' | ';
+
+export default function DataAccessRequestHeader(props) {
+
+  const [datasets] = useState(props.dar.datasets || []);
+  const [consentName] = useState(props.consentName || '');
+  const [darCode] = useState(props.dar.darCode || '');
+  const [projectTitle] = useState(props.dar.projectTitle || '');
+  const [fullDatasets, setFullDatasets] = useState([]);
+
+  // Populate remote datasets here
+  useEffect(() => {
+    // dar.datasets.value holds the ID of the dataset
+    const datasetIds = map('value')(datasets);
+
+    async function fetchDatasets() {
+      const response = await Promise.all(
+        map((id) => {
+          return DataSet.getDataSetsByDatasetId(id);
+        })(datasetIds),
+      );
+      setFullDatasets(response);
```

Comment:
Not necessary, but I'm wondering if we would ever want to sort the datasets to ensure they appear in a consistent pattern for the header. I'm not sure how many datasets are typically named on a DAR for that to be useful though.

Reply:
I really like this idea. Gonna give it a shot

Reply:
Done